src/Symfony/Component/Form/Tests/Fixtures/CustomArrayObject.php

```diff
@@ -29,13 +29,7 @@ public function offsetExists($offset): bool
         return \array_key_exists($offset, $this->array);
     }
 
-    /**
-     * @param mixed $offset
```

Comment:
I did not turn those into native types because this is a test case and other methods would need the type also. This is not worth it.